mllm/models/qwen_npu/modeling_qwen_npu.hpp

```
@@ -0,0 +1,556 @@
+
+  nn::Embedding embedding_;
+
+  void clearKVCache() {
+    for (auto& block : decode_blocks_.list()) { block.getKVCache().clearCache(); }
+  }
+};
+
+class QwenForCausalLM : public nn::Module, public ARGeneration {
+ public:
+  QwenForCausalLM() = default;
+  explicit QwenForCausalLM(const std::string& name, const QwenNPUConfig& cfg) : cfg(cfg), nn::Module(name) {
```

Clang (14.0.6)

[error] constructor does not initialize these fields: lm_head_

(cppcoreguidelines-pro-type-member-init,-warnings-as-errors)